.github/workflows/pr-checks.yml

```diff
@@ -7,14 +7,14 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
`v4` is the latest major version of this action at the time of this pull request's creation.